pkg/controller/datavolume-controller.go

```diff
@@ -278,6 +278,26 @@ func (r *DatavolumeReconciler) Reconcile(req reconcile.Request) (reconcile.Resul
 	return r.reconcileDataVolumeStatus(datavolume, pvc)
 }
 
+func (r *DatavolumeReconciler) findPVCForDV(dataVolumeNamespace string, dataVolumeName string) (*corev1.PersistentVolumeClaim, error) {
+	labelSelector, err := labels.Parse(fmt.Sprintf("cdi-controller=%v", dataVolumeName))
```

Comment:
Note that bartosz's PR is removing this label, so we need to coordinate this.

Reply:
yes, removing, and if we really need this then we need to change the way label is created (directly storing datavolume name will not work for longer names datavolume-253 > label-63), 
maybe look at getPodFromPvc in datavolume-controller, or store uuid-name, or... I do not know

Reply:
Oh no!

If the label is removed, I'd have to list all PVCs in the namespace and find the right one using Owner Reference. Or is there a faster way?

Reply:
Well you can find a way to put there a label that is valid ;), I am not sure if it is a good pattern, but you can for example store string(pvcUID)+"-dv" (like it is done for label CloneUniqueID.

and if not then you should be able to limit pvc to the only cdi managed pvc's? I think we have a label for this.

Reply:
I've given it some thought. I think the cost of calling a k8s api dominates over the cost of processing a list of items. We would make the call in any case, so I'd rather go for a simple code than relying another label.

Let me try that.